File: java/src/ReverseString.java
import java.util.Arrays;

public class ReverseString {
    public void reverseString(char[] s) {
        char temp;
        for (int i = 1; i <= s.length / 2; i++) {
            temp = s[i - 1];
            s[i - 1] = s[s.length - i];
            s[s.length - i] = temp;
        }
        System.out.println(Arrays.toString(s));
    }
}

File: cpp/reverseString.cpp
#include <iostream>
#include <vector>
using namespace std;
void reverseString(vector<char> &s)
{
    reverse(s.begin(), s.end());
}
int main()
{
    vector<char> s = {'h', 'e', 'l', 'l', 'o'};
    reverseString(s);
    for (int i = 0; i < s.size(); i++)
    {
        cout << s[i];
    }
    return 0;
}